apps/web/src/routes/orders/order-list/components/forms/order-form.tsx

```diff
@@ -190,6 +190,11 @@ const OrderSummary = ({ form }: { form: UseFormReturn<OrderFormData> }) => {
 const CustomerSummary = ({ form }: { form: UseFormReturn<OrderFormData> }) => {
   const [isCustomerDialogOpen, openCustomerDialog, closeCustomerDialog] = useToggleState();
   const [selectedCustomer, setSelectedCustomer] = useState<ICustomer | null>(null);
+  const customer = useWatch({ name: 'customer' }) as ICustomer;
+
+  useEffect(() => {
```

Comment:
I think we can the leave useEffect out and pass in the customer directly right? As it gets loaded on mount